Environment_Engine/Query/HasMergeablePropertiesWith.cs

```diff
@@ -0,0 +1,99 @@
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using BH.oM.Environment.Elements;
+using BH.oM.Reflection.Attributes;
+using System.ComponentModel;
+
+using BH.oM.Physical.Constructions;
+using BH.oM.Physical.Materials;
+using BH.Engine.Diffing;
+using BH.oM.Diffing;
+
+namespace BH.Engine.Environment
+{
+    public static partial class Query
+    {
+        [Description("Evaluates if the two elements non-geometrical data is equal to the point that they could be merged into one object. Environment Edges have no additional data other than their geometry, so this will always return true")]
+        [Input("element", "An Environment Edge to compare the properties of with an other Environment Edge")]
+        [Input("other", "The Environment Edge to compare with the other Environment Edge.")]
+        [Output("equal", "True if the Objects non-geometrical property is equal to the point that they could be merged into one object")]
+        public static bool HasMergeablePropertiesWith(Edge element, Edge other)
+        {
+            return true; //Environment Edges have no additional data to be checked so geometric edges can be merged
+        }
+
+        [Description("Evaluates if the two elements non-geometrical data is equal to the point that they could be merged into one object")]
+        [Input("element", "An Environment Panel to compare the properties of with an other Environment Panel")]
+        [Input("other", "The Environment Panel to compare with the other Environment Panel.")]
+        [Output("equal", "True if the Objects non-geometrical property is equal to the point that they could be merged into one object")]
+        public static bool HasMergeablePropertiesWith(Panel element, Panel other)
+        {
+            DiffConfig config = new DiffConfig()
+            {
+                PropertiesToIgnore = new List<string>
+                {
+                    "ExternalEdges",
+                    "Openings",
+                    "ConnectedSpaces",
+                    "Type",
+                },
+                NumericTolerance = BH.oM.Geometry.Tolerance.Distance,
+            };
+
+            string elementDiff = element.DiffingHash(config);
+            string otherDiff = other.DiffingHash(config);
+
+            return elementDiff.Equals(otherDiff);
```

Comment:
After further discussion, should be switched for 
```
DifferentProperties(this IBHoMObject obj1, IBHoMObject obj2, DiffConfig diffConfig = null)
```

and return `true` if the output is `null`